package.json

```diff
@@ -5,46 +5,29 @@
-    "build": "mkdir build; browserify src/index.js -g yo-yoify -o build/app.js -t [ babelify ]; babel --plugins=transform-es2015-template-literals build/app.js --out-file build/app.js",
+    "browser-test-remote-chrome": "env-cmd .env.sauce.local nightwatch --config nightwatch.js --env chrome",
+    "browser-test-remote-firefox": "env-cmd .env.sauce.local nightwatch --config nightwatch.js --env default",
+    "browser-test-remote-ie": "env-cmd .env.sauce.local nightwatch --config nightwatch.js --env ie",
+    "browser-test-remote-safari": "env-cmd .env.sauce.local nightwatch --config nightwatch.js --env safari",
+    "browser-test-remote-parallel": "env-cmd .env.sauce.local nightwatch --config nightwatch.js --env safari,ie,default,chrome",
+    "build": "mkdir -p build; browserify src/index.js -g yo-yoify -t [ babelify --presets [ es2015 ] ] -o build/app.js",
+    "watch": "mkdir -p build; watchify src/index.js -p [ browserify-livereload --host 127.0.0.1 --port 1337 ] -t [ babelify --presets [ es2015 ] ] -o build/app.js -dv",
     "csslint": "csslint --ignore=order-alphabetical --errors='errors,duplicate-properties,empty-rules' --exclude-list='assets/css/font-awesome.min.css' assets/css/",
     "downloadsolc": "rm soljson.js; wget https://ethereum.github.io/solc-bin/soljson.js",
+    "make-mock-compiler": "node ci/makeMockCompiler.js",
     "lint": "standard",
     "serve": "http-server .",
+    "sauce": "env-cmd .env.sauce.local node sc",
     "test": "standard; babel src --out-dir babelify-src; node test/index.js"
   },
   "devDependencies": {
     "async": "^2.1.2",
-    "babel-cli": "^6.16.0",
-    "babel-eslint": "^7.1.1",
-    "babel-plugin-check-es2015-constants": "^6.8.0",
-    "babel-plugin-transform-es2015-arrow-functions": "^6.8.0",
-    "babel-plugin-transform-es2015-block-scoped-functions": "^6.8.0",
-    "babel-plugin-transform-es2015-block-scoping": "^6.18.0",
-    "babel-plugin-transform-es2015-classes": "^6.18.0",
-    "babel-plugin-transform-es2015-computed-properties": "^6.8.0",
-    "babel-plugin-transform-es2015-destructuring": "^6.18.0",
-    "babel-plugin-transform-es2015-duplicate-keys": "^6.8.0",
-    "babel-plugin-transform-es2015-for-of": "^6.18.0",
-    "babel-plugin-transform-es2015-function-name": "^6.9.0",
-    "babel-plugin-transform-es2015-literals": "^6.8.0",
-    "babel-plugin-transform-es2015-object-super": "^6.8.0",
-    "babel-plugin-transform-es2015-parameters": "^6.18.0",
-    "babel-plugin-transform-es2015-shorthand-properties": "^6.18.0",
-    "babel-plugin-transform-es2015-spread": "^6.8.0",
-    "babel-plugin-transform-es2015-sticky-regex": "^6.8.0",
-    "babel-plugin-transform-es2015-template-literals": "^6.8.0",
-    "babel-plugin-transform-es2015-unicode-regex": "^6.11.0",
-    "babel-plugin-transform-regenerator": "^6.16.1",
-    "babel-polyfill": "^6.22.0",
-    "babelify": "^7.3.0",
+    "babel-preset-es2015": "^6.22.0",
```

Comment:
I think we should do all that stuff in another PR
also there were a reason why we didn't use preset but I can't remember now. That might be because of the `async` plugin.